helper/restore_helper.go

```diff
@@ -351,6 +379,10 @@ func doRestoreAgentInternal(restoreHelper IRestoreHelper) error {
 						logWarn(fmt.Sprintf("Oid %d, Batch %d: Skip file discovered, skipping this relation.", tableOid, batchNum))
 						err = nil
 						skipOid = tableOid
+						if *singleDataFile && readers[contentToRestore] != nil {
```

Comment:
```suggestion
						if *singleDataFile && readers[contentToRestore] != nil && readers[contentToRestore].readerType == SUBSET {
```

Reply:
we cannot access `readerType` directly, because it is a private field